### DISCUSSION TITLE
Tablist Sorting

### DISCUSSION THREAD

**Discussion by TarboushMCC** [Zrips/CMI#5233] [Feature Requests]
Could you add a way to sort players in the TAB list based on their permissions, meaning if someone has permission for example tab1, they will be at the top of the tablist, tab2, will be second, etc...

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
